src/Validator/ConstraintsSetFactory/Field/StringFactory.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AssoConnect\ValidatorBundle\Validator\ConstraintsSetFactory\Field;
+
+use Doctrine\DBAL\Types\Types;
+use Symfony\Component\Validator\Constraints\Length;
+use Symfony\Component\Validator\Constraints\NotBlank;
+
+class StringFactory implements FieldConstraintsSetFactoryInterface
+{
+    public function supports(string $type): bool
+    {
+        return Types::STRING === $type;
+    }
+
+    public function getConstraints(array $fieldMapping): array
+    {
+        $constraints = [new Length(['max' => $fieldMapping['length'] ?? 255])];
+
+        if (isset($fieldMapping['nullable']) && true === $fieldMapping['nullable']) {
```

Comment:
Before we add this constraint for all types. I don't know if it's better or not but we change the behavior here.

Reply:
Nope! The constraint was and is still only set for the "string" field
As-is today: https://github.com/assoconnect/validator-bundle/blob/main/src/Validator/Constraints/EntityValidator.php#L233